### DISCUSSION TITLE
Activity disappeared

### DISCUSSION THREAD

**Discussion by Kusieqs** [community/community#114528] [New to GitHub]
My activity on github disappeared (mainly between April and September). Is this some kind of bug? or maybe because the repositories are private and I didn't do anything with them

**Comment by benogoulart** [comment]
Hi,

It's probably not a bug, although it's difficult to say without more information.

If the repositories are **private**, simply not making changes to them **wouldn't cause your past contributions to disappear**. Previous contributions should still be counted (assuming you have the **"Include private contributions on my profile"** option enabled).

Some common reasons for missing contributions include:

- The commits were made with an email address that is no longer associated with your GitHub account.
- The commit author information was changed.
- The repositories or their default branches were modified.
- The contribution graph is no longer able to associate those commits with your account.

Could you clarify a couple of things?

- Are the commits still present in the repositories?
- Did your contribution graph disappear only for **April–September 2023**, or for other periods as well?
- Do you have **"Include private contributions on my profile"** enabled in your profile settings?

That information will help narrow down the possible cause.
